doc/modules/language-guide/pages/modules-and-imports.adoc

```diff
@@ -91,19 +91,21 @@ The call to `+Counters.Counter(1)+` installs a fresh counter on the network. Ins
 
 The type annotation  `+: Counters.Counter+` is redundant here. It's included only to illustrate that the type of the actor class is available when required.
 
-== Importing from another canister
+== Importing from another canister smart contract
 
-In addition to the examples above that import {proglang} modules, you can also import functions from canisters by using the `+canister:+` prefix in place of the `+mo:+` prefix.
+In addition to the examples above that import {proglang} modules, you can also import actors (and their shared functions) from canister smart constracts by using the `+canister:+` prefix in place of the `+mo:+` prefix.
+
+NOTE: Unlike a Motoko library, an imported canister can be implemented in any other {IC} language that emits Candid interfaces for its canister smart contracts (for instance Rust). It could even be an older or newer version of Motoko.
```

Comment:
```suggestion
NOTE: Unlike a {proglang} library, an imported canister can be implemented in any other {IC} language that emits Candid interfaces for its canister smart contracts (for instance Rust). It could even be an older or newer version of {proglang}.
```